### PR TITLE
remote benchmark implementation

### PR DESCRIPTION
**Pull request by aph5nt**

### CODE REVIEW

File: neurons/miners/bitcoin/funds_flow/graph_search.py [hunk 1]
```diff
@@ -51,12 +51,11 @@ def execute_query(self, query: protocol.Query) -> protocol.QueryOutput:
...
         except Exception as e:
             raise Exception(f"cypher query execution error: {e}")
-    
-    
+
     def execute_cypher_query(self, cypher_query: str):
         with self.driver.session() as session:
             result = session.run(cypher_query)
-            return result
+            return result.single()
```
Comment (wendellswa06) [inline neurons/miners/bitcoin/funds_flow/graph_search.py:58]: If memgraph returns multiple results, it should arise warnings. It may cause issue.

Comment (aph5nt) [reply neurons/miners/bitcoin/funds_flow/graph_search.py:63]: ok, i added new method ```execute_benchmark_query```, idea is that benchmat must return single scalar value;
as we use those in Kmeans clustering to determinate majority of results

Comment (aph5nt) [reply neurons/miners/bitcoin/funds_flow/graph_search.py:63]: check new commit

**Review by ronnie-samaroo — COMMENTED** [review]
The PR name is "remove benchmark implementation"
Is it typo? You meant "remote benchmark implementation"?

**Review by wendellswa06 — CHANGES_REQUESTED** [review]

File: neurons/miners/blacklist.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def query_blacklist(self, synapse: protocol.Query) -> typing.Tuple[bool, str]:
                 f"Blacklisting hot key {hotkey} because of wrong model type"
             )
             return True, "Model type not supported."
-        if not is_query_only(synapse.query):
+        if not is_query_only(self.miner_config.query_restricted_keywords ,synapse.query):
```
[the remark this answers is not in the document]
Comment (aph5nt) [reply neurons/miners/blacklist.py:44]: seems it was 2.1.x bug

**Pull request merged**